README.md

```diff
@@ -73,6 +73,20 @@ Description will be shortened to 150 characters where necessary:
 into this
 ![stripped](https://cloud.githubusercontent.com/assets/487758/6201672/27c76794-b4f2-11e4-93d7-96a576285604.png)
 
+#### count(field_name, name='', description='')
+Returns a function that can be used to count many-to-many relationship on a Django admin list view.
+If no name is given, count_*field_name* is used.
+
+**Note** *name* must be a string (not unicode). For translations, use *description* instead
+
+```python
+import admin_methods.list
+
+class UnitGroupAdmin(admin.ModelAdmin):
+  list_display = ('Units',)
+  Units = admin_methods.list.count('units')
```

Comment:
I'm not sure this would work tbh. because the name of the function returned will be `count_units`.